config/initializers/assets.rb

```diff
@@ -0,0 +1,2 @@
+Rails.application.config.assets.precompile += %w(compute_resources/xenserver/cache_refresh.js)
```

Comment:
While this might work, we usually use a different snippet in plugins. See https://github.com/theforeman/foreman_plugin_template/blob/master/lib/foreman_plugin_template/engine.rb#L43